web_search_nospace/static/src/js/search.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * © 2017 Therp BV <http://therp.nl> License AGPL-3.0 or later
+ * (http://www.gnu.org/licenses/agpl.html).
+ */
+openerp.web_search_nospace = function(instance) {
+	instance.web.search.InputView = instance.web.search.InputView.extend({
```

Comment:
don't do this. This way, you actually *replace* the `InputView` object, this can cause very hard to debug problems. Say use [include](https://github.com/OCA/OCB/blob/8.0/addons/web/static/src/js/openerpframework.js#L148) instead, this modifies the existing object